kernel/src/fs/ramfs/memfd.rs

```diff
@@ -98,6 +97,10 @@ impl MemfdInode {
         }
         Ok(())
     }
+
+    pub fn name(&self) -> String {
+        self.name.clone()
```

Comment:
This should return `&String`. The caller can convert it to a `String` if it really needs to do so?